app/plugin.go

```diff
@@ -60,8 +65,16 @@ func (a *App) SyncPluginsActiveState() {
 
 			// Activate plugin if enabled
 			if pluginEnabled {
-				if err := a.Plugins.Activate(pluginId); err != nil {
+				updatedManifest, activated, err := a.Plugins.Activate(pluginId)
```

Comment:
I need the `updatedManifest` here since we compute the bundle hash when it's loaded.